src/Screens/Main/Chat/Title.tsx

```diff
@@ -24,6 +24,7 @@ type Props = OwnProps & DispatchProps & StateProps;
 
 const Title: React.FC<Props> = ({ style, onPress, name, buddyId }) => {
   const color = getBuddyColor(buddyId);
+
   return (
```

Comment:
Nice to have the 'zen' line : D

Reply:
But not really part of this PR, but doesn't matter.

Reply:
Yep, decided to let it be =)